### ISSUE TITLE
C++ desktop api calls is broken by 1.0 preview2 single-project MSIX packaging

### ISSUE THREAD

**Issue by seven-mile** [microsoft/WindowsAppSDK#1564]
**Describe the bug**

Internal bug: https://microsoft.visualstudio.com/OS/_workitems/edit/36569414

I created an initial Desktop **C++**/WinRT WinUI project using the newest template "Blank App, Packaged (WinUI3 in Desktop)", which should be using the preview MSIX self-packaging tooling.

But I noticed that the project name in solution explorer is annotated with suffix (Universal Windows), which is of course wrong. I checked the `.vcxproj` file and found this:

```
<AppContainerApplication>true</AppContainerApplication>
<AppxPackage>true</AppxPackage>
```

But no matter the two switches is on or off, calling functions like `GetWindowsDirectory`, `RegOpenKey` will get a compiler error saying that identifier not found.

The function declarations in WindowsSDK include files was grayed by intellisense, because `#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP | WINAPI_PARTITION_SYSTEM)` is false.

As you can guess, the reason was possibly the desktop partition of API family was disabled.

I don't know where to change this settings, if there won't be any hot fix, could you tell me the workaround?

**Steps to reproduce the bug**

Create a C++ project with the template "Blank App, Packaged (WinUI3 in Desktop)", and call desktop-specific APIs.

**Expected behavior**
Desktop projects should have capabilities to call all native APIs.

**Version Info**
Windows 11 Build 22000.194
Visual Studio 2022 Preview 4.1

WindowsAppSDK 1.0 preview2

| Windows version                    | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              | Yes   |
| May 2020 Update (19041)            |    |
| November 2019 Update (18363)       |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |

**Additional context**

**Comment by andrewleader** [comment]
Hi @seven-mile, this is a known issue in Preview 2 that we're working on fixing: https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/preview-channel#other-limitations-and-known-issues

> C++ template for Visual Studio 2019 and Visual Studio 2022. In this release, these projects are restricted to calling the subset of Win32 APIs that can be called by UWP apps. The Blank App, Packaged with WAP (WinUI 3 in Desktop) template is not affected by this issue.

This only affects the single-project templates that are new in Preview 2. You can instead use the **Blank App, Packaged with WAP (WinUI 3 in Desktop)** template to create a project the old way that includes two projects and it'll work correctly.

I'll leave this bug open to track the status of this issue since we didn't have a public bug tracking it yet (we had internal bugs tracking it).

**Comment by andrewleader** [comment]
Hi @seven-mile, I believe this was fixed in 1.0 Preview 3. It has definitely been fixed in the upcoming 1.0 GA, marking as closed. Thanks!
